Add sriov related functions

Yingshun opened the pull request:
It's required by sriov migration test for now.

Signed-off-by: Yingshun Cui <yicui@redhat.com>

chunfuwen commented:
suggest change python name to sriov_utils.py to keep consistent with other function utils

chunfuwen commented on virttest/sriov.py:
```diff
@@ -0,0 +1,110 @@
+    Check if available PF exists to test
+
+    :param session: The session object to the host
+    :raise: test.cancel when no pf found
```
generally, we don't have test.cancel available, but just exception in utils module

chunfuwen commented on virttest/sriov.py:
```diff
@@ -0,0 +1,110 @@
+    """
+    Check if available PF exists to test
+
+    :param session: The session object to the host
```
miss driver python doc

chunfuwen commented on virttest/sriov.py:
```diff
@@ -0,0 +1,110 @@
+    cmd = "echo %s > %s/sriov_numvfs" % (vf_no, pci_addr)
+    s, o = utils_misc.cmd_status_output(cmd, shell=True,
+                                        verbose=True, session=session)
+    if not s:
```
generally , we need keep one function or method to have one return value in whatever situation if you want them return value. 
So suggest change to :
return not s

Yingshun commented:
> suggest change python name to sriov_utils.py to keep consistent with other function utils

Updated the file to utils_sriov.py and fixed all your comments.
